inspirehep/modules/editor/views.py

```diff
@@ -42,6 +51,56 @@
                       url_prefix='/editor',)
 
 
+@blueprint.route('/revisions/<rec_uuid>/revert', methods=['PUT'])
+def revert_to_revision(rec_uuid):
+    """Revert given record to given revision"""
+    record = Record.get_record(rec_uuid)
+    revision_id = request.json['revision_id']
+    record.revert(revision_id)
+    db.session.commit()
+    return jsonify(success=True)
+
+
+@blueprint.route('/revisions/<recid>', methods=['GET'])
+def get_revisions(recid):
+    """Get revisions of given record"""
+    Transaction = transaction_class(RecordMetadata)
+
+    pid = PersistentIdentifier.query.filter_by(pid_value=recid).one()
+    record = get_db_record(pid.pid_type, recid)
+
+    revisions = []
+    for revision in record.revisions:
+        transaction_id = revision.model.transaction_id
+
+        user_email = Transaction.query.filter(
+            Transaction.id == transaction_id).one().user.email
```

Comment:
can we have a transaction without a user? like first db insert! In that case that would crash.
@jmartinm